tests/browser/affiliations/remove_primary_affiliation.robot

```diff
@@ -33,20 +35,16 @@ Remove Primary Affiliation for Contact2
     Scroll Element Into View  text:Do Not Contact
     Click Button    title:Edit Primary Affiliation
     Wait For Locator  record.edit_form
-    #Sleep    5
-    Scroll Page To Location    100    500
+    Page Scroll To Locator  detail_page.edit_mode.section_header    Contact Information    
```

Comment:
Can you do `Scroll Element Into View    text:Contact Information` so that we don't need to add a locator?

Reply:
that doesn't work as there is Contact Information twice on the page and this one is second one.